README.md

```diff
@@ -58,6 +58,32 @@ log/database/PID files.
 files there. Files in the `etc/` directory of this repository can be used as an
 example (also see below for more about configuration)
 
+Installing automation-tools in Docker
```

Comment:
```suggestion
Running automation-tools in Docker Compose
```

Reply:
I am torn here, could be a discussion in Slack with folks - the management of our environment is done via docker-compose and so it makes the commands intuitive in one sense: `docker-compose exec --user=root archivematica-dashboard bash` whereas, interacting with `docker` works more globally across an environment. If we cut down the steps below to docker-variant vs. generic installation steps above, perhaps notes can be made for both?

Reply:
I made this Docker-specific just because I don't really have time (none of us do) for sorting out the differences. But I can try, and if there are none, I think we should just have one generic installation guide. Most people know how to get into Docker without needing the command. I copy-pasted from my notes as I went along trying to install this on a fresh VM.